index.html

```diff
@@ -198,158 +161,238 @@ <h2>Tonic</h2>
+            </li>
+            <li>
+              <a href="#"
+                ><img class="v-social-media" src="images/vec.png" alt="vector"
+              /></a>
             </li>
           </ul>
-        </form>
+          <button type="button" class="click-btn">Get my resume</button>
+        </section>
+        <section>
+          <ul class="lang">
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Languages</p>
+                <img src="./images/down.png" alt="arrow_bottom" />
+              </div>
+              <ul class="languages">
+                <li>
+                  <img src="images/js.png" alt="JavaScript" />
+                  <a href="#">JavaScript</a>
+                </li>
+                <li>
+                  <img src="images/html..png" alt="CSS" />
+                  <a href="#">HTML</a>
+                </li>
+                <li>
+                  <img src="images/css..png" alt="CSS" />
+                  <a href="#">CSS</a>
+                </li>
+              </ul>
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Frameworks</p>
+                <img src="./images/forward.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Skills</p>
+                <img src="images/forward.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+          </ul>
+        </section>
       </div>
+    </section>
+    <footer>
+      <section class="contact">
+        <div class="contact-content">
+          <h1 class="contact-me">Contact me</h1>
+          <p class="paragraph item">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form
+          class="form-section"
+          action="https://formspree.io/f/mqknjqvv"
+          method="post"
+        >
+          <input
+            id="name"
+            class="text-holder1"
+            type="text"
+            name="name"
+            maxlength="30"
+            required
+            placeholder="Name"
+          />
+          <input
+            id="email"
+            class="text-holder1"
+            type="email"
+            name="name"
+            required
+            placeholder="Email"
+          />
+          <textarea
+            id="message"
+            class="text-area"
+            placeholder="Write your message here"
+            maxlength="500"
+            required
+          ></textarea>
+          <div id="form-button">
+            <button
+              id="clicked-btn"
+              class="click-btn"
+              type="submit"
+              name="sumit"
+            >
```

Comment:
- [ ] As requested by the previous reviewers, kindly make sure that the button is positioned at the center as it is in the chosen Figma design. I have included a comment for you in the [previous reviewer's comment](https://github.com/Hirwacardinalloic/PORTFOLIO/pull/7#discussion_r922699435).